examples/swift_protobuf_example/MODULE.bazel

```diff
@@ -0,0 +1,67 @@
+"""
+Example module for grpc + proto target generation.
+"""
+
+# rules_go
+
+bazel_dep(name = "rules_go", version = "0.50.1")
+
+go_sdk = use_extension("@rules_go//go:extensions.bzl", "go_sdk")
+go_sdk.host()
+
+# gazelle
+
+bazel_dep(name = "gazelle", version = "0.40.0")
+
+# protobuf
+
+bazel_dep(name = "protobuf", version = "29.0")
+
+# rules_proto
+
+bazel_dep(name = "rules_proto", version = "7.0.2")
+
+# apple_support
+
+bazel_dep(name = "apple_support", version = "1.17.1")
+
+# rules_swift
+
+bazel_dep(name = "rules_swift", version = "2.2.2")
+
+# rules_swift_package_manager
+
+bazel_dep(
+    name = "rules_swift_package_manager",
+    version = "0.0.0",
+)
+local_path_override(
+    module_name = "rules_swift_package_manager",
+    path = "../..",
+)
+
+# swift_deps START
```

Comment:
You can remove the `# swift_deps START/END` lines. They are no longer necessary.